app/controllers/use_case_controller.rb

```diff
@@ -41,7 +41,9 @@ def edit
   def update
     @model_instance.assign_attributes(self.permitted_params)   # don't hit db yet
 
-    if self.errors.empty?
+    if @model_instance.changed_attributes.blank? && self.errors.empty?
```

Comment:
Does this still account for a situation where 1) the fields don't change but 2) a different approver is selected?

Unrelated: probably a good idea to add a flash notice